GOVERNANCE.md

```diff
@@ -607,7 +607,7 @@ Role:
 #### Accounts and services ownership and administration
 The PyMC Project also has accounts and hosts services on several platforms.
 Some examples of such platforms include (but are not limited to)
-GitHub, Discourse, PyPI, Discord, Twitter, ReadTheDocs, or Medium.
+GitHub, Discourse, PyPI, Discord, X, ReadTheDocs, or Medium.
```

Comment:
I suggest we close the X account ...

Reply:
@juanitorduz Just removed the X account in GOVERNANCE.md.

Reply:
hi! I meant this beyond this PR ;) Any thoughts @twiecki ?

Reply:
I don't think we should close it.

Reply:
@juanitorduz and @twiecki , I realized that I didn't reply to this PR earlier. I have added back the X (in place of Twitter). Is this PR ready to merge now?